src/components/SettingsItem/index.js

```diff
@@ -0,0 +1,13 @@
+import { CustomInput } from "../shared/CustomInput";
+import styles from "./SettingsItem.module.scss";
+
+export const SettingsItem = ({ type, onChange, title, settingsField }) => (
```

Comment:
ok. your solution was better. return the name props above settingsField

Reply:
What do you mean?

Reply:
settingsField -> name